tfjs-backend-webgpu/src/backend_webgpu.ts

```diff
@@ -51,6 +51,12 @@ type TensorData = {
   shape: number[],
   refCount: number,
   resourceInfo?: BufferInfo|TextureInfo,
+  // zeroCopy is used for creating tensor from GPUBuffer. When zeroCopy is false
+  // or undefined (default), this GPUBuffer will be copied to the tensor's
+  // resource buffer. When zeroCopy is true, tensor will use this GPUBUffer as
+  // tensor's resource buffer, user should not destroy this GPUBuffer until all
+  // access are done.
+  zeroCopy?: boolean,
```

Comment:
I still like `external` as the name to indicate the resource comes from external and can't be destroyed in tfjs.

Reply:
And change `WEBGPU_TENSOR_FROM_BUFFER_WITH_ZERO_COPY` flag to `WEBGPU_TENSOR_USE_EXTERNAL_BUFFER` to keep consistency?